shared/profile/index.js.flow

```diff
@@ -1,11 +1,11 @@
 // @flow
 import {Component} from 'react'
 import * as Types from '../constants/types/profile'
+import type {Response} from 'react-native-image-picker'
```

Comment:
we shouldn't have RN only things in a shared file like this

Reply:
What's the right thing to do? Use `any` instead?

Reply:
thats fine for now, or you could make the actual type yourself (whatever part you use), aka
type Response ={
data: string
} or whatever